woocommerce/wc-admin
# Improve AddFirstProduct email note contents

Fixes https://github.com/woocommerce/woocommerce-admin/issues/6363

This PR improves the contents of AddFirstProduct email note:

- Add merchant's name to greet (with default as "Hi there")
- Update heading
- Update image

It also changes `NotificationEmail` to utilize WC_Email's capability for placeholder contents in email notes by using the `format_string` class function. This allows for dynamic customization and personalization in email notes.

### Screenshots
![image](https://user-images.githubusercontent.com/3747241/111576606-c65d4e00-87eb-11eb-818c-2679e053ef20.png)

### Detailed test instructions:
- Checkout this branch.
- Create a zip for testing with `npm run test:zip`.
- Create a `jurassic.ninja` instance.
- Upload the plugin and activate it.
- Update the installation date (we need a store between 2 and 5 days old). You can do it with an SQL statement like this:

```
UPDATE `wp_options` SET `option_value`=UNIX_TIMESTAMP(DATE_SUB(NOW(), INTERVAL 4 day)) WHERE `option_name` = 'woocommerce_admin_install_timestamp';
```

- Make sure the `woocommerce_merchant_email_notifications` option is set to `yes`:
```
UPDATE `wp_options` SET `option_value` = 'yes' WHERE `wp_options`.`option_name` = 'woocommerce_merchant_email_notifications';
```

- Run the `wc_admin_daily ` cron job (this tool can help [WP Crontrol](https://wordpress.org/plugins/wp-crontrol/)).
- You should have received an email like the image above.

No changelog (landed in 2.2.0)
